docs/docs/concepts.mdx

```diff
@@ -496,35 +501,56 @@ For specifics on how to use retrievers, see the [relevant how-to guides here](/d
 ### Tools
 <span data-heading-keywords="tool,tools"></span>
 
-Tools are interfaces that an agent, a chain, or a chat model / LLM can use to interact with the world.
+Tools are utilities designed to be called by a model: their inputs are designed to be generated by models, and their outputs are designed to be passed back to models.
+Tools are needed whenever you want a model to control parts of your code or call out to external APIs.
+
+A tool consists of:
+
+1. The name of the tool.
+2. A description of what the tool does.
+3. A JSON schema defining the inputs to the tool.
+4. A function (and, optionally, an async variant of the function).
+
+When a tool is bound to a model, the name, description and JSON schema are provided as context to the model.
```

Comment:
```suggestion
When a tool is bound to an LLM, the name, description and JSON schema are provided as context to the model.
```

Reply:
would beef up this section to say "coming up with model name and description should be thought of as the same as prompt eng"

Reply:
mentioned in Best practices, you think should be in both places?

Reply:
I've been trying to use `model` more than `LLM` since we have the concept of an `LLM` as a text-in/text-out model - should I not be?

Reply:
my thinking as well. LLM means something specific in our case